zipkin-server/src/main/resources/zipkin-server-shared.yml

```diff
@@ -90,6 +90,8 @@ zipkin:
       date-separator: ${ES_DATE_SEPARATOR:-}
       index-shards: ${ES_INDEX_SHARDS:5}
       index-replicas: ${ES_INDEX_REPLICAS:1}
+      username: ${ES_USER:}
```

Comment:
I understand why you chose ES_USER/PASS, seeing below that mysql does this. just that mysql is the anomaly (maybe a mistake, but too late to fix as in use).

USERNAME and PASSWORD are the more frequently used, and are nice as they match the property naming conventions.